app/components/conferences/display/EventShow.js

```diff
@@ -59,6 +59,7 @@ export const EventShow = ({ event, error, speaker, prsession }) => {
               <div className={styles.event_logo}>
                 <Image src={event.data.attributes["logo-url"]} width={100} />
               </div>
+              <center><h6> Organizer </h6></center>
```

Comment:
@r3yc0n1c please use `div` with style `text-align=center` in place of `center`, please avoid using `center` since it is already [depreciated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center).

Thank you!